docsite/source/index.html.md

```diff
@@ -188,14 +188,11 @@ Here's a simple usage example how you could access an operation powered by dry-m
 ```ruby
 class UsersController < ApplicationController
   def create
-    resolve("users.create").(safe_params[:user]) do |m|
-      m.success do |user|
-        render json: user
-      end
-
-      m.failure do |code, errors|
-        render json: { code: code, errors: errors.to_h }, status: :unprocessable_entity
-      end
+    case resolve("users.create").(safe_params[:user])
+    in Success[user]
```

Comment:
Returning `user` within an array makes no sense here, since we are creating a single user

Reply:
In a production app me might assume that `Dry::Monads` has been included already, but that will create confusion in a document example.